appveyor.yml

```diff
@@ -44,10 +44,18 @@ before_build:
   - nuget sources add -name core -source https://ci.appveyor.com/nuget/ospsuite-core
   - nuget restore
 
+
+after_build:
+  - cd src\ConsoleApp\bin\Debug
```

Comment:
This is working of course, we will probably want to use a couple of AppVeyor environment variables to find the directory.

There is a rake task that creates the portable and that task gets some env vars

```
  - rake "create_portable_setup[%APPVEYOR_BUILD_VERSION%, %CONFIGURATION%, pk-sim-portable-setup.zip]"
 ```

So, I bet you just want to replace `Debug` with `%CONFIGURATION%`